sympy/integrals/laplace.py

```diff
@@ -1538,6 +1566,25 @@ def _inverse_laplace_time_shift(F, s, t, plane):
     return None
 
 
+def _inverse_laplace_freq_shift(F, s, t, plane):
+    """
+    Helper function for the class InverseLaplaceTransform.
+    """
+    if not F.has(s):
+        return F*DiracDelta(t), S.true
+    if len(args := F.args) == 1:
+        a = Wild('a', exclude=[s])
+        if (ma := args[0].match(s-a)) and re(ma[a]) > 0:
```

Comment:
Testing `if expr > 0` can lead to an exception in the symbolic case:
```
In [1]: if x > 0:
   ...:     print('x > 0')
   ...: 
...
TypeError: cannot determine truth value of Relational
```

Reply:
Potentially `(re(ma[a]) > 0) is True` could be used. I'm not sure how you want to handle this here.

Reply:
Oh, that was one I missed. My next push will have `if (ma := args[0].match(s-a)) and re(ma[a]).is_positive:`